### ISSUE TITLE
Additional swipe options

### ISSUE THREAD

**Issue by eamondaly** [irssiconnectbot/irssiconnectbot#17]
Not sure if this is where you want to see feature requests, but since my phone doesn't have a trackball I'd love to be able to change the left-side up/down swipes from "page up/page down" to "arrow up/arrow down" via a menu option. Similarly, I never switch hosts, so an additional option in the "Swipe Left/Right" menu for "arrow left/arrow right" would be a big win for me.

**Comment by nickurak** [comment]
Maybe swiping up/down in center of the screen would be good for Up/Down? That'd make me really happy.

**Comment by hansfn** [comment]
Yes, swipe gestures for arrow up/down (and left/right) would be really useful.
